pom.xml

```diff
@@ -46,7 +46,7 @@
         <version.testng>6.14.3</version.testng> <!-- aligns with Arquillian TestNG version -->
         <version.junit>4.13.1</version.junit>
         <version.hamcrest>1.3</version.hamcrest>
-        <version.arquillian>1.6.0.Final</version.arquillian>
+        <version.arquillian>1.7.0.Alpha9</version.arquillian>
```

Comment:
Did you check the junit and testng versions? They should align with the ones transitively added by Arquillian.

Reply:
@radcortez I checked the testng 6.14.3 works with this version. The transitive dependency on junit brought in by Arquillian 1.7.0.Alpha9 is 4.13.1, so this PR is good.

Reply:
Hum, but it seems that Arquillian updated the TestNG version:
https://github.com/arquillian/arquillian-core/blob/master/build/pom.xml#L31

This will create a misalignment between the versions. And it's even more concerning since it's a major version change. I recommend to align the versions used by Arquillian.

Reply:
Interesting. For CDI 3.0 tck tests, I checked testng 6.14.3 worked ok with the arquillian 1.7.0.Alpha9. Anyway, to stay current with testng, I have updated the testng version @radcortez